pkg/e2e/operators/rbac.go

```diff
@@ -50,6 +52,9 @@ var _ = ginkgo.Describe(rbacOperatorBlocking, func() {
 var _ = ginkgo.Describe(subjectPermissionsTestName, func() {
 	h := helper.New()
 	checkSubjectPermissions(h, "dedicated-admins")
+	checkSubjectPermissions(h, "sre-admins")
```

Comment:
Are `checkSubjectpernission` and `daCreateSubjectpernission` Spelling errors? I see that it's a separately-defined function, but it's so similarly named that I did a double-take. Could this be made clearer?